### ISSUE TITLE
protontricks-launch does not work correctly

### ISSUE THREAD

**Issue by free-storage** [Matoking/protontricks#261]
I ran this command `protontricks-launch --no-bwrap --appid <appid> <exe path>`. 

But it opened the exe file in its own container and not in the game container. I need to open the exe in the container with the game as this command does it `/home/deck/.steam/steam/steamapps/common/SteamLinuxRuntime_sniper/pressure-vessel/bin/steam-runtime-launch-client --bus-name=com.steampowered.App<appid> -- wine <exe path>`
![image](https://github.com/Matoking/protontricks/assets/52704675/de11dbe5-dbd7-4e95-aa3f-ca1233da04ec)

**Comment by Matoking** [comment]
The windows in the taskbar denote which applications are related (i.e. they have the same program name). See the following screenshot and the "Group" setting which governs this (you can access this by right-clicking the taskbar and selecting _Configure Icons-only Task Manager..._):

![icon-task-manager-settings](https://github.com/Matoking/protontricks/assets/959667/285dc552-fbf0-4ff9-970e-24f34e64e0a6)

Containers are a different mechanism and governs what parts of the system the applications are able to access. For example, the Dolphin file manager is able to access pretty much everything your user has access to, whereas Steam games run through Steam Runtime are more restricted.

`protontricks-launch` with `--no-bwrap` was able to access the game's memory the last time I tried and the application appearing in a different part of the taskbar should not affect this. Does this not work in your case for some reason?

**Comment by free-storage** [comment]
> `protontricks-launch` with `--no-bwrap` was able to access the game's memory the last time I tried and the application appearing in a different part of the taskbar should not affect this. Does this not work in your case for some reason?

Yes. Does not work. The trainer does not see the running game. I think it's because they are in different containers. How to launch in one container, similar to `.../steam-runtime-launch-client --bus-name=com.steampowered.App<appid> -- wine <exe path>` ?

**Comment by Matoking** [comment]
Closing old issue, please open a new issue if this is still a problem.
